cmd/store/create.go

```diff
@@ -46,7 +46,7 @@ func create(fgaClient client.SdkClient, storeName string) (*client.ClientCreateS
 	return store, nil
 }
 
-func createStoreWithModel(
+func CreateStoreWithModel(
```

Comment:
If a store ID is provided, we should not be creating the store, just updating the existing store name, model and appending the tuples

Reply:
I thought so but couldn't find the ID mentioned here so I thought the StoreId wouldn't be present in a store file - https://github.com/openfga/cli/blob/main/internal/storetest/storedata.go#L58.

Is it present though? Is there any spec I can see on how the store file can be structured?

Reply:
The id should be passed in as a flag or read from env.

Like how GetStore does